src/main/resources/jupyter/jupyter_notebook_config.py

```diff
@@ -28,7 +28,7 @@
 if os.environ['WELDER_ENABLED'] == 'true':
   # Only enable this along with Welder, as this change is backwards incompatible
   # for older localization users.
-  c.NotebookApp.notebook_dir = '/home/jupyter-user/notebooks'
+  c.NotebookApp.notebook_dir = ${NOTEBOOKS_DIR}
```

Comment:
In order to use it here I think `NOTEBOOKS_DIR` needs to be in the `environment` section of `jupyter-docker-compose.yaml`'. Something like:
```
NOTEBOOKS_DIR: "${NOTEBOOKS_DIR}"
``

Reply:
Also this may need to be
```
os.environ['NOTEBOOKS_DIR']
```,
similar to the above evs.